released/packages/coq-autosubst-ocaml/coq-autosubst-ocaml.1.0.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "A code generator for substitution boilerplate"
```

Comment:
```suggestion
synopsis: "A Coq plugin that generates variable substitution boilerplate code"
```
This is currently a very generic synopsis, maybe you can be a bit more specific here?